appinventor/appengine/build.xml

```diff
@@ -379,6 +379,7 @@
         <pathelement location="${lib.dir}/guava/guava-18.0.jar" />
         <pathelement location="${lib.dir}/gwt_dragdrop/gwt-dnd-3.2.3.jar" />
         <pathelement location="${lib.dir}/gwt_incubator/gwt-incubator-20101117-r1766.jar" />
+        <pathelement location="${lib.dir}/moxieapps/org.moxieapps.gwt.uploader-1.1.0.jar" />
```

Comment:
It's from Moxie Group (http://www.moxiegroup.com/moxieapps/gwt-uploader/). The Uploader is open-source, released in 2012 and updated then. I used it because it makes handling files simpler and can be built upon. Do you think having old code is a big enough issue to warrant holding off on this feature?